lib/rubycritic/command_factory.rb

```diff
@@ -4,9 +4,18 @@ module Rubycritic
   class CommandFactory
     def self.create(options = {})
       Config.set(options)
+      validate_rails_directory if Config.for_rails
       command_class(Config.mode).new(options)
     end
 
+    def self.validate_rails_directory
+      raise ArgumentError, 'The option -r should be executed inside a rails app root folder' unless is_a_rails_app?
+    end
+
+    def self.is_a_rails_app?
+      File.exists?('Gemfile')
```

Comment:
I'd say 90% of all modern non-rails-but-ruby have a Gemfile in their root directory so this check is actually a bug. I'd rather check for things that are only rails specific OR not care at all since checking this might always lead to edge case situations which will trip users up.